server/src/og.ts

```diff
@@ -96,31 +99,35 @@ const getOGPageSubstack = async (subdomain: string, sld: string, landingPageSett
   }
 
   const { data } = await substackAxiosBase.get(`https://${url.host}/${path}`)
-  if (domainInfo.farcastEnabled) {
-    const jsdom = new JSDOM(data)
-    const image = jsdom.window.document.querySelector('meta[name="og:image"]')?.getAttribute('content') ?? undefined
-    let farcasterPartial: string | undefined
-    let farcasterFull: string | undefined
-    if (domainInfo.farcastBasicMap) {
-      farcasterPartial = renderFarcasterMapBasicPartialTemplate(domainInfo, image)
-    } else if (domainInfo.farcastMap) {
-      farcasterFull = await renderFarcasterMapFullTemplate(domainInfo)
-      farcasterPartial = ''
-    } else if (domainInfo.farcastText) {
-      farcasterPartial = renderFarcasterTextTemplate(domainInfo, image)
-    } else {
-      farcasterPartial = renderFarcasterPartialTemplate(domainInfo, image)
-    }
+  const jsdom = new JSDOM(data)
+  const image = jsdom.window.document.querySelector('meta[name="og:image"]')?.getAttribute('content') ?? undefined
+  let farcasterPartial: string
+
+  if (domainInfo.farcastBasicMap) {
+    farcasterPartial = renderFarcasterMapBasicPartialTemplate(domainInfo, image)
+  } else if (domainInfo.farcastMap) {
+    const farcasterFull = await renderFarcasterMapFullTemplate(domainInfo)
+    farcasterPartial = ''
     if (farcasterFull) {
       return farcasterFull
     }
-    const partialDom = JSDOM.fragment(farcasterPartial)
-    jsdom.window.document.head.append(partialDom)
-    return jsdom.serialize()
+  } else if (domainInfo.farcastText) {
+    farcasterPartial = renderFarcasterTextTemplate(domainInfo, image)
+  } else if (domainInfo.farcastPartial) {
+    farcasterPartial = renderFarcasterPartialTemplate(domainInfo, image)
+  } else if (domainInfo.farcastSwap) {
+    const vwap = 0 // TODO: get $b price here
```

Comment:
same as this - not much to do with OG or substack

Reply:
Done